libraries/CoreAudio/CoreAudio.cpp

```diff
@@ -230,7 +239,9 @@ void CoreAudio::action( int id ) {
         {
           if (soundPlayFlashRaw.isPlaying())
           {
-            soundPlayFlashRaw.stop();
+            //soundPlayFlashRaw.stop();
```

Comment:
please remove the commented line. It's fine to have the previous version in the history and a cleaner code